build_tools/github_actions/post_build_upload.py

```diff
@@ -40,8 +41,11 @@ def log(*args):
     sys.stdout.flush()
 
 
-def exec(cmd: list[str], cwd: Path):
-    log(f"++ Exec [{cwd}]$ {shlex.join(cmd)}")
+def exec(cmd: list[str], cwd: Path, useShlexCmdLogging: bool = True):
+    if useShlexCmdLogging:
+        log(f"++ Exec [{cwd}]$ {shlex.join(cmd)}")
+    else:
+        log(f"++ Exec [{cwd}]$ {' '.join(cmd)}")
     subprocess.run(cmd, check=True)
```

Comment:
This extra argument seems unnecessary. Why not always do the same logging? How about adding a separate helper function for running powershell commands that directly does what it needs, instead of abstracting behind this _logging_ argument? In either case, the comment block about logging should be moved out of `upload_artifacts()` - none of that is relevant to file uploading.

If keeping the argument, use `underscore_case`: https://peps.python.org/pep-0008/#function-and-variable-names